modules/convert/notification.go

```diff
@@ -48,12 +48,18 @@ func ToNotificationThread(n *models.Notification) *api.NotificationThread {
 				result.Subject.LatestCommentURL = comment.APIURL()
 			}
 		}
-	case models.NotificationSourceCommit:
+	case models.NotificationSourceCommit: //unused until now
```

Comment:
Don't think this comment is helpful or enlightening

```suggestion
	case models.NotificationSourceCommit:
```